style.css

```diff
@@ -9,10 +9,38 @@
  * minimal changes to the HTML itself.
  */
 
+ header {
+ 	background-color: #000066;
+ 	color: white;
+ 	font-family: Courier New, Courier, monospace;
+ 	letter-spacing: 3px;
+ 	text-align: center;
+ 	padding-top: 10px;
+    padding-right: auto;
+    padding-bottom: 10px;
+    padding-left: auto;
+}
+code:hover {
+	-webkit-transform: rotate(360deg);
```

Comment:
The transform on mine doesn't work... There's a couple of things going on here:

1. the vendor prefix `-webkit-` means it will only work for webkit based browsers (chrome, safari but not firefox)
2. When using a prefix its worth looking at sites like [Can I Use](http://caniuse.com/#search=transform) which will display a little icon if a prefix is necessary. In this case `transform` alone would work for more people.
3. I had a little investigation and found that transform transitions don't work on `inline` elements. To get around this you can add `display: inline-block;` and then the transition and rotation both work as expected.